src/views/Player/Player.test.js

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Player from '../../views/Player/Player';
+
+test('renders correctly when there are selections', () => {
+  const container = render(
```

Comment:
This test should be async with an `await findyByText` to properly this the component and avoid a memory leak